src/createServer.js

```diff
@@ -1,3 +1,39 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('../src/convertToCase/convertToCase');
+const { validation } = require('./validation');
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const { pathname, searchParams } = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = pathname.slice(1);
+    const targetCase = searchParams.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = validation(res, originalText, targetCase);
+
+    if (errors.length) {
+      res.statusCode = 400;
```

Comment:
Consider sending a descriptive error message along with the 400 status code to provide more information about the error.